CameraExtrinsicMat:
   rows: 4
   cols: 4
   dt: d
   data: [ 1.9656451260809171e-02, 6.0112648566115479e-02,
       9.9799804278675719e-01, 1.1148978024721146e-01,
       -9.9965768894571294e-01, -1.6056848799106127e-02,
       2.0656295368706723e-02, 2.2147584706544876e-02,
       1.7266408299007663e-02, -9.9806244648769826e-01,
       5.9776450676186976e-02, -1.4754268527030945e-01, 0., 0., 0., 1. ]
CameraMat:
   rows: 3
   cols: 3
   dt: d
   data: [ 2.4032691757255579e+03, 0., 1.2166054602770371e+03, 0.,
       2.4121512925696502e+03, 1.0082126790786290e+03, 0., 0., 1. ]
DistCoeff:
   rows: 1
   cols: 5
   dt: d
   data: [ -1.7891839184119299e-01, 1.5709201128078301e-01,
       7.5643992933044482e-05, -2.0919911715141070e-04, 0. ]
DistModel: plumb_bob
ImageSize: [ 2448, 2048 ]
T_Sensor2Cam:
   rows: 3
   cols: 1
   dt: d
   data: [ 1.1970042374413857e-01, 2.2073899464143017e-02,
       -1.4097475278460703e-01 ]
R_Sensor2Cam: [ 2.0694777369499207e-02, -5.9812106192111969e-02,
    -1.7298202961683273e-02 ]
T_Cam2Optical:
   rows: 3
   cols: 1
   dt: d
   data: [ 0., 0., 0. ]
R_Cam2Optical: [ -1.5707963705062866e+00, 0., -1.5707963705062866e+00 ]
img_pts:
   rows: 76
   cols: 1
   dt: "2f"
   data: [ 2293., 1910., 2297., 1230., 1616., 1872., 1613., 1224., 1478.,
       1862., 850., 1849., 1488., 1224., 838., 1222., 93., 1912., 773.,
       1858., 754., 1217., 759., 500., 759., 1195., 116., 1188., 175.,
       481., 731., 1186., 1407., 1207., 752., 510., 1417., 525., 1663.,
       1203., 1647., 527., 2316., 523., 1853., 1678., 1824., 1182.,
       2308., 1159., 2329., 1662., 1040., 1163., 1524., 1169., 1518.,
       1655., 1038., 1647., 550., 1155., 129., 1165., 560., 1616., 131.,
       1582., 565., 1238., 145., 1232., 164., 800., 575., 811., 1005.,
       825., 1011., 1278., 1472., 1270., 1451., 830., 1853., 852., 2304.,
       834., 2333., 1295., 1870., 1301., 2408., 1641., 2404., 1263.,
       2022., 1263., 1534., 1251., 1903., 1270., 1882., 1647., 1520.,
       1628., 848., 1263., 1217., 1255., 859., 1637., 1228., 1637., 379.,
       1591., 18., 1591., 381., 1240., 31., 1242., 37., 911., 14., 1270.,
       369., 1286., 387., 921., 654., 930., 656., 1290., 1013., 1286.,
       1017., 928., 1772., 942., 1411., 938., 1770., 1309., 1405., 1303.,
       2058., 938., 2412., 951., 2041., 1301. ]
lidar_pts:
   rows: 76
   cols: 1
   dt: "3f"
   data: [ 2.39676809e+00, -9.79660392e-01, -8.75066102e-01,
       2.40276814e+00, -9.68979359e-01, -1.89046681e-01, 2.52414846e+00,
       -3.31249565e-01, -8.58913958e-01, 2.53792500e+00, -3.30911785e-01,
       -2.06437409e-01, 2.59131527e+00, -2.13106424e-01, -8.79089653e-01,
       2.60380602e+00, 4.36360180e-01, -8.93752158e-01, 2.55022407e+00,
       -2.06370845e-01, -2.01898634e-01, 2.55006075e+00, 4.42947179e-01,
       -2.16815531e-01, 2.35645962e+00, 1.12209332e+00, -8.79527926e-01,
       2.50486422e+00, 4.91186410e-01, -8.63024116e-01, 2.43711376e+00,
       5.00563323e-01, -2.08670750e-01, 2.46933198e+00, 5.10569036e-01,
       5.07649899e-01, 2.43046188e+00, 4.92658287e-01, -1.88977137e-01,
       2.41224742e+00, 1.13525701e+00, -1.95421353e-01, 2.37032342e+00,
       1.08175004e+00, 5.21251321e-01, 2.42955995e+00, 5.24215341e-01,
       -1.92720070e-01, 2.44837976e+00, -1.26940623e-01, -1.91484585e-01,
       2.42235136e+00, 5.15037239e-01, 4.95310545e-01, 2.49619555e+00,
       -1.33107871e-01, 4.95337248e-01, 2.41418910e+00, -3.55724126e-01,
       -1.80293277e-01, 2.41957331e+00, -3.51095051e-01, 4.86034453e-01,
       2.38222265e+00, -9.89671111e-01, 5.17371476e-01, 3.30632067e+00,
       -7.56211936e-01, -8.26450706e-01, 3.33073211e+00, -7.27428794e-01,
       -1.61693692e-01, 3.31250119e+00, -1.38889492e+00, -1.17559552e-01,
       3.26345229e+00, -1.38174307e+00, -7.95686126e-01, 3.39337635e+00,
       3.18070710e-01, -1.56225085e-01, 3.40035200e+00, -3.33348244e-01,
       -1.46011472e-01, 3.36297560e+00, -3.27811867e-01, -8.26241851e-01,
       3.36964703e+00, 3.22130531e-01, -8.26403260e-01, 3.48181462e+00,
       1.01507342e+00, -1.50524974e-01, 3.51840520e+00, 1.66684198e+00,
       -1.90424800e-01, 3.46458578e+00, 1.00078976e+00, -8.12512755e-01,
       3.45576143e+00, 1.64619493e+00, -7.95055270e-01, 3.47181654e+00,
       9.96755660e-01, -2.81521440e-01, 3.50001860e+00, 1.63476157e+00,
       -2.94534564e-01, 3.57813430e+00, 1.66068172e+00, 3.66746068e-01,
       3.53793144e+00, 1.02015483e+00, 3.43726277e-01, 3.56682301e+00,
       3.96633685e-01, 3.31784129e-01, 3.56828690e+00, 3.81172150e-01,
       -3.26407552e-01, 3.50545597e+00, -2.72288978e-01, -2.98737407e-01,
       3.56687903e+00, -2.34184369e-01, 3.30552936e-01, 3.57717156e+00,
       -8.23509634e-01, 3.43029141e-01, 3.48090672e+00, -1.46772051e+00,
       3.48167062e-01, 3.41565132e+00, -1.47000504e+00, -3.15288663e-01,
       3.56791162e+00, -8.51498544e-01, -3.40376735e-01, 4.16619253e+00,
       -1.93672907e+00, -9.56019998e-01, 4.20456886e+00, -1.95406950e+00,
       -3.03472400e-01, 4.40257597e+00, -1.33919191e+00, -3.06299567e-01,
       4.29539394e+00, -4.46230978e-01, -2.96940684e-01, 4.30931377e+00,
       -1.09982359e+00, -3.17862630e-01, 4.26917171e+00, -1.04277515e+00,
       -9.92956996e-01, 4.27796602e+00, -4.11595643e-01, -9.74505782e-01,
       4.31900549e+00, 7.46844411e-01, -3.51824164e-01, 4.28890848e+00,
       9.96759906e-02, -3.10499787e-01, 4.30580759e+00, 7.34476388e-01,
       -9.89484668e-01, 4.24892139e+00, 7.74887279e-02, -9.61392283e-01,
       4.48156977e+00, 1.65866351e+00, -9.96450782e-01, 4.39184713e+00,
       2.32148194e+00, -1.03572023e+00, 4.52206182e+00, 1.66732466e+00,
       -3.38471770e-01, 4.42890263e+00, 2.32346487e+00, -3.59675288e-01,
       4.29954481e+00, 2.25714350e+00, 2.79292226e-01, 4.29813576e+00,
       2.29071045e+00, -4.11014915e-01, 4.44093943e+00, 1.66375983e+00,
       -4.27748084e-01, 4.45662212e+00, 1.64584446e+00, 2.71457314e-01,
       4.39391327e+00, 1.12193966e+00, 2.61885285e-01, 4.42994547e+00,
       1.11163008e+00, -4.08320785e-01, 4.41882181e+00, 4.59938765e-01,
       -3.97814155e-01, 4.39713001e+00, 4.49056715e-01, 2.55179048e-01,
       4.38221788e+00, -8.88928473e-01, 2.56989598e-01, 4.41930819e+00,
       -2.36205190e-01, 2.57216573e-01, 4.43213081e+00, -8.84367466e-01,
       -4.06516910e-01, 4.48012304e+00, -2.34433919e-01, -4.12801147e-01,
       4.38941097e+00, -1.40585101e+00, 2.66416430e-01, 4.33054066e+00,
       -2.03284621e+00, 2.73454547e-01, 4.40052605e+00, -1.37219596e+00,
       -3.83913875e-01 ]
ReprojectionError: 1.5895942703943156e+00
